src/server_manager/web_app/ui_components/outline-share-dialog.ts

```diff
@@ -125,8 +125,8 @@ Polymer({
           <li>
             [[localize('share-invite-instructions')]]
             <a
-              href="https://github.com/Jigsaw-Code/outline-client/blob/master/docs/invitation-instructions.md"
-              >https://github.com/Jigsaw-Code/outline-client/blob/master/docs/invitation-instructions.md</a
+              href="https://github.com/Jigsaw-Code/outline-client/blob/master/docs/invitation_instructions.md"
```

Comment:
Actually, can you duplicate the file in the old location (maybe with a symlink)? I wanna make sure the existing clients keep working.